TanoaPeakSimulator/TanoaPeakSimulator/Tanoa.py

```diff
@@ -0,0 +1,128 @@
+import secrets
+import random
+
+
+def main():
+    location_length = 1000
+    bp_length = 100
+    bp_amount = 200
+    # This should be in a function with the file as a parameter for when this gets turned into
+    genome_file = open("dmel-all-chromosome-r6.38.fasta", "r")  # code which takes arguments
+    sequences = genome_file.read()
+    sequences_arr = sequences.split('>')
+    sequence_object_array = []
+    for sequence in sequences_arr:
+        sequence_split = sequence.splitlines()
+        if len(sequence_split) != 0:
+            header = sequence_split[0]
+            body = ""
+            for x in range(1, len(sequence_split)):
+                body += sequence_split[x]
+            sequence_object_array.append(Chromosome(header, body))
+
+    # Code for running all of the chromosomes. Commented out for now since I don't have enough memory...
+    # for chromosome in sequence_object_array:
+    #     for x in range(2000):
+    #         get_random_base_pair_string(location_length, bp_length, chromosome.sequence,
+    #                                     chromosome.bp_segment_arr)
+    # for chromosome in sequence_object_array:
+    #     chromosome.small_bp_segment_list = get_random_base_pair_string_list(chromosome.bp_segment_arr, bp_length,
+    #                                                                         bp_amount, chromosome.sequence)
+    chromosome = sequence_object_array[1]
+    for x in range(100):
+        get_random_base_pair_string(location_length, bp_length, chromosome.sequence, chromosome.bp_segment_arr)
+
+    chromosome.small_bp_segment_list = get_random_base_pair_string_list(chromosome.bp_segment_arr, bp_length,
+                                                                        bp_amount, chromosome.sequence)
+
+    generate_output_files(chromosome)
+
+
+# Gets a psudo-random string of a given amount of characters from the main sequence. This will move to taking
+# the buffers on the sides and saving that as well so when sampled later we don't have to return to the
+# Very large string. That's inefficient.
+# Parmeters: sequence_length: Length of the big sample that you want to take
```

Comment:
Is this different from just doing `len(sequence)`?  Or by big sample, do you mean the size of the enhancer regions?

Reply:
yea, sequence_length is the length of the enhancer region. 
I'll also look into doc strings
And for looping over single line, you want individual files per each chromosome? As is now it will output _all_ of the chromosomes in the same file if you make the code run through them. 
And I'll look into generators. I've never heard of those, so I'm not sure if I should go through with changing all of the code now or after this PR and v 0.1 is stable

Reply:
I'm okay with all enhancer regions in one file and all fake reads in another file.  Don't need separate files for each chromosome.

Reply:
ok on the generator thing. And on the line by line thing, if I call the file.write(myString) multiple times without it overwriting what was there then I can do the line by line thing and just add each chromosome as I go. I was under the impression I can only write once and then it'd overwrite the file. I'm guessing I can only "open" once and write as many times as I want?

Reply:
You can call `write()` as many times as you want.  If you need to reopen a file to append to it, you can use `open(flname, "a")`  (See documentation [here](https://docs.python.org/3/library/functions.html#open).)

Reply:
k, I'll change the import then so it uses the with open('data.fasta') as f thingy and see if that helps me allow it to just run the entire file. I'll try implementing that and some other changes monday and you'll see how it went by tuesday I guess :)

Reply:
Sounds great.  Thanks for the hard work!